app-utils/edid-decode/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.1+git20230804
+SRCS="git::commit=5f723267e04deb3aa9610483514a02bcee10d9c2::https://git.linuxtv.org/edid-decode.git"
+CHKSUMS="sha256::c10823c940641404cd6eb9c801d43b307e87e9252e8eded66eed8be750c4f884"
```

Comment:
Missing CHKUPDATE - use Anitya.

Reply:
The package has no release tags, and does not appear on https://release-monitoring.org/

Reply:
Got it, please leave a `# FIXME:` comment.